scipy/stats/tests/test_discrete_basic.py

```diff
@@ -271,3 +271,30 @@ def test_methods_with_lists(method, distname, args):
     npt.assert_allclose(result,
                         [dist.pmf(*v) for v in zip(z, *p2, loc)],
                         rtol=1e-15, atol=1e-15)
+
+
+@pytest.mark.parametrize(
+    'dist, x, args',
+    [
+        (stats.hypergeom, np.arange(5), (3, 3, 4)),
+        (stats.nhypergeom, np.arange(-2, 5), (5, 2, 8)),
```

Comment:
Do we need to have different `x` ranges for each?

Reply:
For some it's necessary but for most of them, the range of (0, 5) is enough. (I checked with all the distributions and most of them failed in that range).